src/main/java/com/lambdaschool/foundation/models/PopulationHist.java

```diff
@@ -0,0 +1,80 @@
+package com.lambdaschool.foundation.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "populationhist")
+public class PopulationHist extends Auditable
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long popid;
+
+    @NotNull
+    private int year;
+
+    @NotNull
+    private double pop;
+
+    @ManyToOne
+    @JoinColumn(name = "cityid")
+    @NotNull
+    @JsonIgnoreProperties(value = "populationhist")
+    private City city;
+
+    public PopulationHist()
+    {
+    }
+
+    public PopulationHist(
+        @NotNull int year,
+        @NotNull double pop,
+        @NotNull City city)
+    {
+        this.year = year;
+        this.pop = pop;
+        this.city = city;
+    }
+
+    public long getPopid()
+    {
+        return popid;
+    }
+
+    public void setPopid(long popid)
```

Comment:
Generated id

Reply:
While we are generating the id, we still want to have the ability to change it if we want. Because the id is being generated locally on our backend, the ID isn't a parameter in any of the constructors. Being that the Id field is private we want to retain the ability to update it through the setter